source/TilegramData.js

```diff
@@ -0,0 +1,22 @@
+import pitchPopulationTilegram from '../tilegrams/pitch-us-population-500k.json'
+
+class TilegramData {
+  constructor() {
+    this._tilegrams = [
+      {
+        label: 'Pitch U.S. Population 2016',
```

Comment:
Yeah I guess it makes sense since the map was made this year. I'll make a trello tag to check for updated USCensus data (right now it's from July 2015, which was the most recent when I grabbed it a few months ago). I think there was a big census data release the other day though.